docs/reference/ERROR_CODES.md

```
@@ -334,6 +334,17 @@ Error: CBKR201_RDW_READ_ERROR at record 100
 Failed to read RDW header: Unexpected end of file
 ```
 
+#### CBKR202_RDW_WRITE_ERROR
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)